### DISCUSSION TITLE
My allsky

### DISCUSSION THREAD

**Discussion by barrettepp** [aaronwmorris/indi-allsky#1154] [Show and tell]
I just wanted to show this off.  I have been working on this off and one for the past month.  I cannot post on the facebook page so I wanted to show this here.  

I hope you enjoy.  I still need to work on some of the shades i made to block some exterior lights.   

I am also working on the best way to access/share my videos from off site.  

![IMG_5770](https://github.com/aaronwmorris/indi-allsky/assets/156556984/13ad4f56-2d64-4740-9116-39eb41e4fc5e)

![testing](https://github.com/aaronwmorris/indi-allsky/assets/156556984/13f57cd4-73d4-49cb-81b6-b035ce293d34)

![testing2](https://github.com/aaronwmorris/indi-allsky/assets/156556984/fb914700-65ac-425b-b5f0-5326b4740f98)

**Comment by aaronwmorris** [comment]
That looks pretty awesome!  I can tell the sensor is the imx464 (or 462) because there is so much color in the stars.  The data from those sensors experience a lot of saturation.
